src/userobject/ElasticRecoilCrossSectionUserObject.C

```diff
@@ -49,23 +49,61 @@ ElasticRecoilCrossSectionUserObject::ElasticRecoilCrossSectionUserObject(const I
     _recoil_energy_limits(getParam<std::vector<Real>>("recoil_energy_limits")),
     _file_name(getParam<std::string>("output_file_name"))
 {
+
   // See "enum_order.h and enum_quadrature_type.h"
   _quadrature = QBase::build(QGAUSS, 1, FORTYTHIRD).release();
-
+  /*
   // Extract 1 dim quadrature rule from libmesh and store in points and weights
   const std::vector<Point> & temp_points = _quadrature->get_points();
   for (auto & pt : temp_points)
     _quad_points.push_back(pt(0));
-
   _quad_weights = _quadrature->get_weights();
+  */
+
+  _quad_points =   {-0.999714, -0.998492, -0.996295, -0.993125, -0.988984, -0.983878,
```

Comment:
We should find a nicer place for this than here....not sure where yet, but we can discuss it.